tst/com/amazon/ion/benchmark/DataGeneratorTest.java

```diff
@@ -76,6 +80,9 @@ public class DataGeneratorTest {
     private final static String HEAP_USAGE = "Heap usage";
     private final static String SERIALIZED_SIZE = "Serialized size";
     private final static String SPEED = "speed";
+    private final static File[] TEST_ISL_FILES = new File("./tst/com/amazon/ion/datagenerator/").listFiles();
+    private final static String BEFORE = "testSeed0.ion";
+    private final static String AFTER = "testSeed1.ion";
```

Comment:
Looks like these constants aren't needed; they can be local to the `generateAndCompare` method.